### ISSUE TITLE
Can use python?

### ISSUE THREAD

**Issue by missbook520** [YuzhouPeng/unet-tensorrt#7]
Thank you very much for your work, can I use python to call it?

**Comment by YuzhouPeng** [comment]
Yes, you need to compile code to a library (.so file) and use pybind11 to call the function. Maybe I will add plugin in the future if I have time :)

**Comment by missbook520** [comment]
Thank you very much for your reply. I followed your steps when compiling this project on jeston nano, but the following error was reported when using unet -s. What is the problem? My CUDA version is 10.2, tensorrt version is 7.1.3-1, thank you very much
![38b6ea7cd23228a846a9a65a60f089b](https://user-images.githubusercontent.com/48777650/117540115-262fe280-b040-11eb-9efc-7d2119638178.png)

**Comment by YuzhouPeng** [comment]
你有使用链接里的pytorch unet训练权重吗？我看了下好像weights翻倍了

**Comment by YuzhouPeng** [comment]
如果训练按照步骤来的话，可能是factor的问题，我抽时间fix下把

**Comment by YuzhouPeng** [comment]
https://github.com/wang-xinyu/tensorrtx/tree/master/unet please use newest repo for testing, this old repo is no longer maintained
